README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### A Little Bit About Me!
+
+My name is Lauren Park. I am a second year at the College, and am a Sociology and French major. I was born and raised in Los Angeles. I am also on the women's varsity tennis team. 
+
+#### These are a few of *my favorite things* to do in Chicago:
```

Comment:
You could just do the Italicized *My Favorite Things to do in Chicago* as the header (instead of having it be more of a sentence.